app/src/main/java/com/kickstarter/libs/FragmentViewModel.java

```diff
@@ -24,11 +24,11 @@
 
   private final PublishSubject<ActivityResult> activityResult = PublishSubject.create();
   private final PublishSubject<Bundle> arguments = PublishSubject.create();
-  protected final Koala lake;
+  protected final AnalyticEvents lake;
```

Comment:
nit: Maybe we should change the name of this variable from `lake` to `analyticsEvents`.

Reply:
I'll rename it on a separate PR just with that change, it will affect a bunch of VM's